modules/ingestor/src/graph/sbom/spdx.rs

```diff
@@ -138,24 +138,16 @@ impl SbomContext {
 
         let mut licenses = LicenseCreator::new();
 
-        let license_refs = sbom_data
-            .other_licensing_information_detected
-            .iter()
-            .map(|e| (e.license_identifier.clone(), e.license_name.clone()))
-            .collect::<HashMap<_, _>>();
-
         let mut packages =
             PackageCreator::with_capacity(self.sbom.sbom_id, sbom_data.package_information.len());
 
         for package in &sbom_data.package_information {
             let declared_license_info = package.declared_license.as_ref().map(|e| LicenseInfo {
                 license: e.to_string(),
-                refs: license_refs.clone(),
```

Comment:
If there's a license ref being used in the declared license (or concluded license), shouldn't that ref then be replaced with the license name?

Reply:
Where does that replacement show up in the data model?